marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.tsx

```diff
@@ -129,9 +128,22 @@ const QueryModal = (props) => {
                 query = selectedQuery;
             }
         })
+
+        let arrayProperties = new Array();
+        props.entityDefArray && props.entityDefArray.forEach(entity => {
+            if (entity.name === query.savedQuery.query.entityTypeIds[0]) {
+                entity.properties && entity.properties.forEach(prop => {
+                    if (prop.ref.length === 0 && prop.datatype === 'array') {
+                        arrayProperties.push(prop.name)
+                    }
+                });
```

Comment:
I was asking for what this code block is for, so I can comment on what type of tests we should add.

Reply:
This just looking for array datatype in the properties. I don't think comments are needed for the simple functions.